.github/workflows/render-rmarkdown.yaml

```diff
@@ -0,0 +1,48 @@
+on:
+  push:
+    paths:
+      - 'README.Rmd'
+
+name: Render README
+
+jobs:
+  build:
+    runs-on: macOS-latest
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Setup R
+        uses: r-lib/actions/setup-r@v1
+
+      - name: Install pandoc
+        run: |
+          brew install pandoc
+
+      - name: Cache Renv packages
```

Comment:
With respect to the missing Rmarkdown file, I do think the reason is we do not use an `renv` in the wateRinfo package and do not have an `renv.lock` file available to 'restore' in the next step. 

I would just install the required dependencies for the conversion (or maybe install the package as it has the required dependencies) instead of introducing renv just for this utility